administration/management/system-management/README.md

```diff
@@ -41,10 +41,10 @@ The `ncs.conf` file is described in the NSO [Manual Pages](https://developer.cis
 # ncs -c /etc/ncs/ncs.conf
 ```
 
-However, in a System Install, the `init` script must be used to start NSO, and it will pass the appropriate options to the `ncs` command. Thus NSO is started with the command:
+However, in a System Install, the systemd is typically used to start NSO, and it will pass the appropriate options to the `ncs` command. Thus NSO is started with the command:
```

Comment:
```suggestion
However, in a System Install, the `systemd` is typically used to start NSO, and it will pass the appropriate options to the `ncs` command. Thus NSO is started with the command:
```